java/api/src/main/java/io/ray/api/options/PlacementGroupCreationOptions.java

```diff
@@ -0,0 +1,103 @@
+package io.ray.api.options;
+
+import com.google.common.base.Preconditions;
+import io.ray.api.Ray;
+import io.ray.api.placementgroup.PlacementStrategy;
+import java.util.List;
+import java.util.Map;
+
+/** The options for creating placement group. */
+public class PlacementGroupCreationOptions {
+  public final boolean global;
+  public final String name;
+  public final List<Map<String, Double>> bundles;
+  public final PlacementStrategy strategy;
+
+  public PlacementGroupCreationOptions(
+      boolean global, String name, List<Map<String, Double>> bundles, PlacementStrategy strategy) {
+    Preconditions.checkNotNull(
+        bundles, "`Bundles` must be specified when create a new placement group.");
+    boolean bundleResourceValid =
```

Comment:
Does the python API also do this check? Why not check it in the C++ code?

Reply:
yeah, Currently, Python API will do the same check, maybe we can move it to the core worker in another independent pr if needed?

Reply:
SGTM